lib/rules/no-invalid-position-at-import-rule/__tests__/index.js

```diff
@@ -113,6 +113,59 @@ testRule({
 	],
 });
 
+testRule({
+	ruleName,
+	config: [true, { ignoreAtRules: ['bar', '/^my-/', /foo/] }],
+
+	accept: [
+		{
+			code: stripIndent`
+				@bar 'bar.css';
+				@import 'foo.css';
+			`,
+			description: 'ignored atRule at the beggining - equal string',
+		},
+		{
+			code: stripIndent`
+				@foofoo 'bar.css';
+				@import 'foo.css';
+			`,
+			description: 'ignored atRule at the beggining - match regExp',
+		},
+		{
+			code: stripIndent`
+				@my-custom-rule {};
+				@import 'foo.css';
+			`,
+			description: 'ignored atRule at the beggining - match regExp',
+		},
+	],
+
+	reject: [
+		{
+			code: stripIndent`
+				@baz 'bar.css';
+				@import 'foo.css';
+			`,
+			message: messages.rejected,
+			description: 'atRule does not match the pattern',
+			line: 2,
+			column: 1,
+		},
+		{
+			code: stripIndent`
+				@use 'bar.css';
```

Comment:
```suggestion
				@bar 'bar.css';
```

Sorry, I missed one.